infra/default-tags.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ServiceName": "ADP",
+    "ServiceCode" :"CDO"
```

Comment:
this file exists already doesn't it in the main branch?

Reply:
It previously was not there until Asad completed his PR.

Reply:
Rebased the branch